autopilot/financial-services/bankruptcy.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>use dict comprehension? (also, seems there would be a more efficient and maybe idiomatic way other than <code>iterrows()</code>)</p><pre class="ql-syntax" spellcheck="false">name_mapping = {
    row[0]: row[1]
    for _, row in feature_names.iterrows()
}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
fixed.realized didn't need name_mapping after all